embedder/tests/test_worker.py

```diff
@@ -17,8 +17,18 @@ async def test_embed_item_callback(elasticsearch_storage_handler, embed_queue_it
     unembedded_chunk = elasticsearch_storage_handler.read_item(embed_queue_item.chunk_uuid, "Chunk")
     assert unembedded_chunk.embedding is None
 
-    async with TestRedisBroker(broker) as br, TestApp(app):
+    async with TestRedisBroker(router.broker) as br:
```

Comment:
Is there a `TestRedisRouter` we could use to keep in step with how the embedder works?

Reply:
unfortunately not :(

Reply:
Ah thanks for the docs :)